src/components/breadcrumbs/breadcrumbs.stories.mdx

```diff
@@ -43,6 +44,12 @@ import { Breadcrumbs, Crumb } from "carbon-react/lib/components/breadcrumbs";
 
 ### Breadcrumbs
 
+<StyledSystemProps
+  of={Breadcrumbs}
```

Comment:
```suggestion
```

Running the storybook docs shows the breadcrumbs props table twice, and the reason appears to be this line. Removing it still shows the padding/margin props but without duplicating the other props.

Reply:
It's the ArgsTable below actually, will get it removed

Reply:
well yeah, the `of` produces it once and the `ArgsTable` does the other, you can remove either